TestSuites/FileServer/src/SMB2Model/TestSuite/ReplayCreateDurableHandleV2PersistentTestCase.cs

```diff
@@ -19875,6 +19875,7 @@ public partial class ReplayCreateDurableHandleV2PersistentTestCase : PtfTestClas
         #endregion
         
         #region Test Starting in S1675
+        [Microsoft.VisualStudio.TestTools.UnitTesting.Ignore]
```

Comment:
Those cases are not good for testing Replay. A tempary solution is to remove them from the test suite.